docs/data/sdks/typescript-browser/migration.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Migration Guide
```

Comment:
Can you be a little more clear with the title, so someone searching this would be able to tell what it's for?

Reply:
```suggestion
title: Browser SDK Migration Guide
```